test/bdd/fixtures/scripts/strapi_configure.sh

```diff
@@ -52,6 +52,11 @@ GENERATE_CCS_COMMAND="strapi generate:api creditcardstatements UserID:string VcM
 
 $GENERATE_CCS_COMMAND
 
+# generate the drivinglicenses and model
```

Comment:
Not sure if you meant for driving license to be one word here (but maybe you did based on the variable name)